Postman Test Suite/Ed-Fi ODS-API ChangeQueries Key Changes and Deletes Test Suite.postman_collection.json

```diff
@@ -12764,11 +12764,9 @@
 					"const correlationId = pm.variables.replaceIn(\"{{$randomMACAddress}}\").replace(/:/g, '');",
 					"pm.request.addQueryParams(`correlationId=${correlationId}`);",
 					"",
-					"CreateAccessToken(\"TokenExpiry_255901\",\"AccessToken_255901\",\"ApiKey_255901\",\"ApiSecret_255901\")",
-					"",
-					"CreateAccessToken(\"TokenExpiry_Other_Namespace\",\"AccessToken_Other_Namespace\",\"ApiKey_Other_Namespace\",\"ApiSecret_Other_Namespace\")",
-					"",
-					"CreateAccessToken(\"TokenExpiry_255901044\",\"AccessToken_255901044\",\"ApiKey_255901044\",\"ApiSecret_255901044\")",
+					"CreateAccessToken(\"255901\");",
```

Comment:
I have aligned all the Postman collections with the same `CreateAccessToken` function.